### ISSUE TITLE
Feature Request/Idea: Favicons for browser windows

### ISSUE THREAD

**Issue by donaldguy** [AdamWagner/stackline#89]
This is enough of a big ask that I hesitate to post as an issue rather than just trying to skip to PR (I am blocked on pursuit by a thoroughly unshaved yak of wanting to complete [teal-language/teal-types](https://github.com/teal-language/teal-types) for hammerspoon and/or a script to generate them before writing more hs lua)

But just to throw it out there in case anyone else is interested in beating me to it it / its easier than I'm thinking, it would be nice to have the option to have browser windows under `appearance.showIcons` use the favicon of the foreground tab rather than the browser icon

(I'd not be shocked if there is some browser extension that changes the Dock Icon thus and could do the rest for free; 

or  stackline could just snoop the url (sneakily, or more simply by e.g. requiring it to be making it to the window title first – something I know there are browser extensions and/or native settings to do) and sideload the favicon itself using `hs.http` )
